.taskcluster.yml

```diff
@@ -87,38 +87,67 @@ tasks:
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
       - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
         then:
+          taskId: {$eval: as_slugid("build_release")}
           dependencies:
             - {$eval: as_slugid("lint_task")}
             - {$eval: as_slugid("test_task")}
-          scopes:
-            - secrets:get:project/relman/dump_syms/deploy
           created: {$fromNow: ''}
           deadline: {$fromNow: '2 hour'}
           provisionerId: proj-relman
           workerType: ci
           payload:
-            features:
-              taskclusterProxy:
-                true
             maxRunTime: 3600
             image: "rust:buster"
             env:
               PROJECT_NAME: dump_syms
             command:
               - "/bin/bash"
               - "-cx"
-              - "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py &&
-                 python3 get-pip.py &&
-                 rustup toolchain install stable &&
+              - "rustup toolchain install stable &&
```

Comment:
I wonder if this is actually needed, given that we are in a Rust image.

Reply:
Definitely not, i was able to build with just a `cargo build --release` in the `rust:buster` image. I'm removing that.